### ISSUE TITLE
Command Failed

### ISSUE THREAD

**Issue by shadowcodex** [evertton/grunt-mrdoc#7]
running grunt mrdoc with the following options

```
mrdoc: {
    all: {
        src: 'src',
        target: 'docs',
        options: {
            // task specific options
        }
    }
},
```

results in the following error

```shell
Running "mrdoc:all" (mrdoc) task
>> ERROR: Error: Command failed: /bin/sh -c /home/ubuntu/workspace/node_modules/node_modules/.bin/mr-doc --source src --output docs
>> /bin/sh: 1: /home/ubuntu/workspace/node_modules/node_modules/.bin/mr-doc: not found
```

**Comment by shadowcodex** [comment]
I wonder why it has node_modules twice.

**Comment by shadowcodex** [comment]
fixed with pull_request #8
